save-cloud-common/src/jvmMain/kotlin/com/saveourtool/save/listeners/DateListener.kt

```diff
@@ -0,0 +1,23 @@
+package com.saveourtool.save.listeners
+
+import com.saveourtool.save.entities.vulnerabilities.Vulnerability
+import java.time.LocalDateTime
+import javax.persistence.PrePersist
+import javax.persistence.PreUpdate
+
+@Suppress("MISSING_KDOC_TOP_LEVEL")
+class DateListener {
+    @PrePersist
+    private fun beforeSave(entity: Vulnerability) {
+        with(entity) {
+            createDate = LocalDateTime.now()
```

Comment:
a small suggestion to use a single `LocalDateTime.now()` to set both values, just avoid a possible delay between them

Reply:
Fixed